doc/articles/features/vscode-advanced-mobile-debugging.md

```diff
@@ -12,7 +12,7 @@ Starting from Uno 4.8.26, the Uno Platform solution templates contains the appro
 
 Both [Source Link](https://github.com/dotnet/designs/blob/main/accepted/2020/diagnostics/source-link.md) and "Sources embedded inside PDB" features are used by Uno Platform and are supported by the extension.
 
-However, only Android includes the `.pdb` of the referenced assemblies inside debug builds for net6.0. This makes the features unusable for iOS, macOS, and Mac Catalyst, see [issue](https://github.com/xamarin/xamarin-macios/issues/11879#issuecomment-1279452073).
+However, only Android includes the `.pdb` of the referenced assemblies inside debug builds for net6.0. This makes the features unusable for iOS and macOS, see [issue](https://github.com/xamarin/xamarin-macios/issues/11879#issuecomment-1279452073).

```

Comment:
That was true for `netX.X-macos` but I don't recall (need to double check) this for `netX.X` that is used for Skia/macOS desktop.